idl/reconcilable_result.idl.hh

```diff
@@ -20,12 +20,14 @@
  */
 
 class partition {
-    uint32_t row_count();
+    uint32_t row_count_low_bits();
     frozen_mutation mut();
+    std::optional<uint32_t> row_count_high_bits() [[version 4.2]];
 };
 
 class reconcilable_result {
-    uint32_t row_count();
+    uint32_t row_count_low_bits();
     utils::chunked_vector<partition> partitions();
     query::short_read is_short_read() [[version 1.6]] = query::short_read::no;
+    std::optional<uint32_t> row_count_high_bits() [[version 4.2]];
```

Comment:
I think @avikivity requested change to 4.3, no?

Reply:
yes of course, for some reason I missed it